code/modules/jobs/job_types/_job.dm

```diff
@@ -191,6 +191,11 @@
 		else
 			back = backpack //Department backpack
 
+	if(H.jumps == SKIRT)
```

Comment:
Possible bug here if the outfit datum is re-used, but I think that's not common

Reply:
Not sure when this would be the case, but I've tested each occupation on spawn, on display for the pref screen, and when setting the uniform through admin command. What would cause the bug here if re-used?

Reply:
I think it's fine given how these are supposed to be used, but basically if you equipped the same instance of the datum to two humans in a row, if the first human had a skirt pref then the second human would also get that pref